cmd/endpoint-ellipses.go

```diff
@@ -66,8 +66,12 @@ var isValidSetSize = func(count uint64) bool {
 }
 
 func commonSetDriveCount(divisibleSize uint64, setCounts []uint64) (setSize uint64) {
+	sort.Slice(setCounts, func(i, j int) bool {
+		return setCounts[i] < setCounts[j]
+	})
+
```

Comment:
how is it clean if you remove comments? @shichanglin5

Reply:
It is easy to understand this code based on this context, I think [Comments rules](https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules) is good

If you think the comment is necessary, I can add the comment

Reply:
Yes please.

Reply:
okay